src/MLOps.NET/Entities/Impl/DataColumn.cs

```diff
@@ -44,5 +45,10 @@ public DataColumn(Guid dataSchemaId)
         /// Type
         /// </summary>
         public string Type { get; set; }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public List<DataDistribution> Distribution { get; set; } = new List<DataDistribution>();
```

Comment:
If a `Distribution` is not owned by a `DataColumn` then this makes sense, otherwise we should remove it